test/sql/test_profile/R/test_profile

```diff
@@ -1,3 +1,21 @@
+-- name: test_explain_analyze
+function: prepare_data("tpcds", "${db[0]}")
+-- result:
+None
+-- !result
+shell: env mysql_cmd="${mysql_cmd} -D${db[0]}" bash ${root_path}/sql/test_profile/T/test_profile_analysis.sh
+-- result:
+0
+Analyze profile succeeded
+Analyze profile succeeded
+Analyze profile succeeded
+-- !result
+shell: env mysql_cmd="${mysql_cmd} -D${db[0]}" bash ${root_path}/sql/test_profile/T/test_profile_non_default_variables.sh
+-- result:
+0
+NonDefaultSessionVariables contains 'pipeline_dop'
```

Comment:
why not show its current value? for example  16(default) ->  2(now)

Reply:
this is the output of the shell script, the output of the profile will be checked at the script.

```sh
function test_non_default_variables() {
    analyze_output=$(${mysql_cmd} -e "$1")
    if grep -q "pipeline_dop: 0 -> 2" <<< "${analyze_output}"; then
        echo "NonDefaultSessionVariables contains 'pipeline_dop'"
    else
        echo "NonDefaultSessionVariables not contains 'pipeline_dop'"
        exit 1
    fi
}
```